mikrotik/resource_interface_veth_test.go

```diff
@@ -0,0 +1,213 @@
+					resource.TestCheckResourceAttr(resourceName, "comment", comment),
+					resource.TestCheckResourceAttr(resourceName, "disabled", disabled),
+				),
+			},
+			{
+				Config: testAccInterfaceVeth(name, updatedAddress, updatedGateway, comment),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					testAccInterfaceVethExists(resourceName),
+					resource.TestCheckResourceAttr(resourceName, "name", name),
+					resource.TestCheckResourceAttr(resourceName, "address", updatedAddress),
+					resource.TestCheckResourceAttr(resourceName, "gateway", updatedGateway),
+					resource.TestCheckResourceAttr(resourceName, "comment", comment),
+					resource.TestCheckResourceAttr(resourceName, "disabled", disabled),
+				),
+			},
+			{
+				Config: testAccInterfaceVeth(name, address, gateway, updatedComment),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					testAccInterfaceVethExists(resourceName),
+					resource.TestCheckResourceAttr(resourceName, "name", name),
+					resource.TestCheckResourceAttr(resourceName, "address", address),
+					resource.TestCheckResourceAttr(resourceName, "gateway", gateway),
+					resource.TestCheckResourceAttr(resourceName, "comment", updatedComment),
+					resource.TestCheckResourceAttr(resourceName, "disabled", disabled),
+				),
+			},
+			{
+				Config: testAccInterfaceVethUpdatedDisabled(name, address, gateway, comment, updatedDisabled),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					testAccInterfaceVethExists(resourceName),
+					resource.TestCheckResourceAttr(resourceName, "name", name),
+					resource.TestCheckResourceAttr(resourceName, "address", address),
+					resource.TestCheckResourceAttr(resourceName, "gateway", gateway),
+					resource.TestCheckResourceAttr(resourceName, "comment", comment),
+					resource.TestCheckResourceAttr(resourceName, "disabled", updatedDisabled),
+				),
+			},
+		},
+	})
+}
+
+func TestAccMikrotikInterfaceVeth_import(t *testing.T) {
+	client.SkipIfRouterOSV6OrEarlier(t, sysResources)
+	name := acctest.RandomWithPrefix("tf-acc-import")
+	address := "192.168.88.15/24"
+	gateway := "192.168.88.1"
+	comment := acctest.RandomWithPrefix("tf-acc-comment")
+	
+	resourceName := "mikrotik_interface_veth.bar"
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:                 func() { testAccPreCheck(t) },
+		ProtoV5ProviderFactories: testAccProtoV5ProviderFactories,
+		CheckDestroy:             testAccCheckMikrotikInterfaceVethDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccInterfaceVeth(name, address, gateway, comment),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					testAccInterfaceVethExists(resourceName),
+					resource.TestCheckResourceAttrSet(resourceName, "id"),
+					resource.TestCheckResourceAttrSet(resourceName, "name"),
+				),
+			},
+			{
+				ResourceName: resourceName,
+				ImportState:  true,
+				ImportStateIdFunc: func(s *terraform.State) (string, error) {
+					return name, nil
+				},
```

Comment:
[optional] it can be simplified as
```suggestion
				ImportStateId: name,
```